### ISSUE TITLE
Cannot display the user dashboard (500 due to date conversion issue)

### ISSUE THREAD

**Issue by kaiyou** [pixelfed/pixelfed#453]
The issue appeared right after https://github.com/pixelfed/pixelfed/commit/4ea205491ce30547e0e525464ebd32aeb5d36391 was merged.

I get a 500 everytime I try to open the uesr dashboard as an administrator, the internal error is due to carbon trying to parse an invalid date.

```
[2018-09-09 10:44:12] production.ERROR: DateTime::__construct(): Failed to parse time string (20180733) at position 7 (3): Unexpected character {"userId":1,"email":"xxx@yyy.zz","exception":"[object] (Exception(code: 0): DateTime::__construct(): Failed to parse time string (20180733) at position 7 (3): Unexpected character at /var/www/vendor/nesbot/carbon/src/Carbon/Carbon.php:439)
```

It appears the culprit is https://github.com/pixelfed/pixelfed/blob/eb641d25b426b9d2cbddf998f2e23e8f2d7758d4/app/Http/Controllers/AdminController.php#L100. Indeed, averaging dates stored as ``YYYYMMDD`` will undoubtedly return something that is not guaranteed to be a valid date. It seems to me either I missed a migration here and my ``created_at`` field is stored in the wrong format, or averaging dates is not as wise as averaging the time deltas.

**Comment by dansup** [comment]
@kaiyou Are you using MySQL?

**Comment by kaiyou** [comment]
I am using the Docker ``_/mysql`` container, here is the version string.

```
@ead6547f465f:/# mysqld -V
mysqld  Ver 5.7.23 for Linux on x86_64 (MySQL Community Server (GPL))
```

**Comment by dansup** [comment]
@kaiyou Fixed in 0f673d7
